src/components/buttons/OulineButton.tsx

```diff
@@ -21,7 +21,7 @@ const ButtonContainer = styled.button<IButtonContainerProps>`
 		switch (props.buttonType) {
 			case 'primary':
 				return props.disabled
-					? `color: ${brandColors.giv[500]};border-color: ${brandColors.giv[500]};opacity: 0.4;`
+					? `color: ${brandColors.pinky[300]};border-color: ${brandColors.pinky[300]};`
```

Comment:
Is it based on the design?

Reply:
when button is pinky, it's obvious that its disabled state should be something like pinky not purple. my changes are for primary theme.